src/black/trans.py

```diff
@@ -71,6 +73,8 @@ class CannotTransform(Exception):
 TResult = Result[T, CannotTransform]  # (T)ransform Result
 TMatchResult = TResult[Index]
 
+SPLIT_SAFE_CHARS = frozenset(["\u3001", "\u3002", "\uff0c"])  # East Asian stops
```

Comment:
I see the motivation for this change but it feels separate from the width change, so it should be called out separately in the changelog.

Reply:
Indeed.  I added one more changelog for this.